### PR TITLE
Excelsior Fixes

### PR DESCRIPTION
**Pull request by NanakoAC**
Fixes excelsior implants not being properly setup when spawned by storyteller, resulting in their own turrets shooting them
Fixes storyteller having a defunct "roleset" option in the roleset pool which wasted antag points
Adds many parts to the excelsior disk so they can actually create shield generators

### CODE REVIEW

File: code/game/objects/items/weapons/implant/implant.dm [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 //return TRUE for implanter icon update.
 /obj/item/weapon/implant/proc/install(var/mob/living/target, var/organ, var/mob/user)
+	world << "Implant installing"
```
Comment (mattroks101) [inline code/game/objects/items/weapons/implant/implant.dm]: These look like leftover debug messages? Did you mean to leave these here?

Comment (NanakoAC) [reply code/game/objects/items/weapons/implant/implant.dm]: i did not, good catch

File: code/game/objects/items/weapons/implant/implant.dm [hunk 2]
```diff
@@ -42,14 +42,17 @@
 
 		if(!affected)
 			user << SPAN_WARNING("[H] is missing that body part!.")
+			world << "Fail missing bodypart"
```
Comment (mattroks101) [inline code/game/objects/items/weapons/implant/implant.dm]: One still here.

File: code/game/objects/items/weapons/implant/implant.dm [hunk 3]
```diff
@@ -42,14 +42,17 @@
...
 			return
 
 		if(allowed_organs && allowed_organs.len && !(organ in allowed_organs))
 			user << SPAN_WARNING("[src] cannot be implanted in this limb.")
+			world << "Fail not allowed"
```
Comment (mattroks101) [inline code/game/objects/items/weapons/implant/implant.dm]: And here.

File: code/game/objects/items/weapons/implant/implant.dm [hunk 4]
```diff
@@ -42,14 +42,17 @@
...
 			return
 
 	if(!can_install(target, affected))
 		user << SPAN_WARNING("You can't install [src].")
+		world << "Fail caninstall"
```
Comment (mattroks101) [inline code/game/objects/items/weapons/implant/implant.dm]: And here.

File: code/game/objects/items/weapons/implant/implant.dm [hunk 5]
```diff
@@ -61,6 +64,7 @@
 
 	on_install(target, affected)
 	wearer.update_implants()
+	world << "success"
```
Comment (mattroks101) [inline code/game/objects/items/weapons/implant/implant.dm]: And here.

**Pull request merged**